infrastructure/auth.template

```diff
@@ -0,0 +1,88 @@
+{
+	"AWSTemplateFormatVersion": "2010-09-09",
+	"Description": "Template for creating resources related to authentication",
+	"Metadata": {},
+	"Parameters": {
+		"EmailARN": {
+			"Type": "String",
+			"Default": "arn:aws:ses:us-west-2:006316662763:identity/no-reply@passr.ca",
+			"Description": "SES Email Arn"
+		}
+	},
+	"Mappings": {},
+	"Conditions": {},
+	"Resources": {
+		"DevUserPool": {
+			"Type": "AWS::Cognito::UserPool",
+			"Properties": {
+				"AccountRecoverySetting": {
+					"RecoveryMechanisms": [
+						{
+							"Name": "verified_email",
+							"Priority": 1
+						}
+					]
+				},
+				"AutoVerifiedAttributes": [
+					"email"
+				],
+				"EmailConfiguration": {
+					"EmailSendingAccount": "DEVELOPER",
+					"From": "no-reply@passr.ca",
+					"ReplyToEmailAddress": "help@passr.ca",
+					"SourceArn": {
+						"Ref": "EmailARN"
+					}
+				},
+				"Policies": {
+					"PasswordPolicy": {
+						"MinimumLength": 8,
+						"RequireLowercase": true,
+						"RequireUppercase": true,
+						"TemporaryPasswordValidityDays": 7
+					}
+				},
+				"UsernameAttributes": [
+					"email"
+				],
+				"VerificationMessageTemplate": {
+					"DefaultEmailOption": "CONFIRM_WITH_LINK",
+					"EmailMessage": "Verify your email using this code: {####}\n\n If you need any assistance verifying your email, please contant help@passr.ca",
+					"EmailSubject": "Passr | Verify your email address",
+					"EmailMessageByLink": "Verify your email using this link: {##passr##}\n\n If you need any assistance verifying your email, please contant help@passr.ca",
+					"EmailSubjectByLink": "Passr | Verify your email address",
+					"SmsMessage": "Your Passr Verification Code: {####}"
```

Comment:
Just a thought for way in the future, but adding verification through phone number might be good since I feel like a lot of users would be on mobile!

Reply:
The reason I didn't add it right now is because I thought people would be more comfortable giving an email rather than a phone #, but yeah I think this might be a good idea. For now, emails are a tad bit easier to handle.